model/src/main/java/org/cbioportal/genome_nexus/model/TranscriptConsequence.java

```diff
@@ -70,6 +70,21 @@ public class TranscriptConsequence
 
     private Map<String, Object> dynamicProps;
 
+    private AlphaMissense alphaMissense;
+
+    @Field("alphamissense")
+    public AlphaMissense getAlphaMissense() {
+        return alphaMissense;
+    }
+
+    public void setAlphaMissense(AlphaMissense alphaMissense) {
+        this.alphaMissense = alphaMissense;
+    }
+
+    public void setDynamicProps(Map<String, Object> dynamicProps) {
```

Comment:
Do we need this?

Reply:
<img width="571" alt="image" src="https://github.com/genome-nexus/genome-nexus/assets/114964497/c3e2a834-0553-4197-8f3a-d288aaf278f4">
It is used in VariantAnnotationSummaryServiceImpl.java. I think we need it

Reply:
@JunheZoooo 👆